ONSdigital/ras-frontstage
# RAS-814: Integrate Google Analytics GA4 with RASMRM Frontstage

# What and why?
As part of the migration and integration with google analytics (GA4),As part of the migration and integration with google analytics (ga4), 3 new partial templates have been added that will allow integration with GA4 and incorporates nonce encoding.

# How to test?
Deploy to preprod and check that the analytics are reporting correctly with no errors.

# Jira
https://jira.ons.gov.uk/browse/RAS-814
